config.go

```diff
@@ -1147,92 +1146,6 @@ func cleanAndExpandPath(path string) string {
 	return filepath.Clean(os.ExpandEnv(path))
 }
 
-// parseAndSetDebugLevels attempts to parse the specified debug level and set
```

Comment:
nit: maybe hard while still having every commit compile, but nice if the code can be deleted in the same commit as it is moved. Easier to see what is new vs moved code then :)

Reply:
Makes sense. I commented out some stuff with interim TODO comments that are removed later but I got it to compile on each commit.

Reply:
If code must be commented out for it to compile, then maybe it's better to do move and delete in separates commit (I can just use the total file diff to see what was moved), but ok for now :)

Rationale is for each commit to be self-contained and "working", such that `git bisect` can function properly.